src/Angor/Client/Pages/Create.razor

```diff
@@ -223,334 +228,587 @@
-            @if (creatProjectSpinner)
-            {
-                <span class=" spinner-border spinner-border-sm me-2" role="status" aria-hidden="true"></span>
-                <span>Create Project...</span>
-            }
-            else
-            {
-                <span>Create Project</span>
-            }
-        </button>
-        <div class="card card-body mt-4">
+        <div class="info-card">
+            <h5>Current Step</h5>
+            <div class="text-center">
+                @GetProgressMessage()
+            </div>
+
+            <div class="action-buttons d-flex justify-content-center item-center mt-4">
+                <button class="btn btn-border-success mt-4" @onclick="ConfirmFinalizeNostrProfileCreation" disabled="@nostrMetadataCreated">
+                    @if (createProfileSpinner)
+                    {
+                        <span class=" spinner-border spinner-border-sm me-2" role="status" aria-hidden="true"></span>
+                        <span>Creating Nostr Profile...</span>
+                    }
+                    else
+                    {
+                        <span>Create Nostr Profile</span>
+                    }
+                </button>
+
+                <button class="btn btn-border-success mt-4" @onclick="ConfirmFinalizeProjectInfoCreation" disabled="@(!nostrMetadataCreated || nostrApplicationSpecificDataCreated)">
+                    @if (createApplicationDataSpinner)
+                    {
+                        <span class=" spinner-border spinner-border-sm me-2" role="status" aria-hidden="true"></span>
+                        <span>Creating Project Info...</span>
+                    }
+                    else
+                    {
+                        <span>Create Project Info</span>
+                    }
+                </button>
+
+                <button type="button" class="btn btn-border-success mt-4" @onclick="CreatProjectTransactionAnCheckPassword" disabled="@(creatProjectSpinner || !nostrApplicationSpecificDataCreated || !nostrMetadataCreated)">
+                    @if (creatProjectSpinner)
+                    {
+                        <span class=" spinner-border spinner-border-sm me-2" role="status" aria-hidden="true"></span>
+                        <span>Create Project...</span>
+                    }
+                    else
+                    {
+                        <span>Create Project</span>
+                    }
+                </button>
+            </div>
+        </div>
+
+        <div class="info-card mt-4">
             <div class="d-flex justify-content-between">
                 <button type="button" class="btn btn-border-success" @onclick="() => SetActiveTab(2)">
                     <span>Previous</span>
                 </button>
             </div>
         </div>
+
+        @code {
+        private IEnumerable<(string IconName, string Label, string Value)> GetProjectInfoData()
```

Comment:
Why thus style of coding? does it not make it harder to see how the page is formatted?

Reply:
I did this to avoid duplicate code.

Reply:
hmm ok @DavidGershony what do you think about this ui code?
I think I prefer duplicates because it is more clear what the ui does but maybe I just need to get used to it